DirectProgramming/C++SYCL_FPGA/Tutorials/Features/max_reinvocation_delay/src/max_reinvocation_delay.cpp

```diff
@@ -69,7 +69,7 @@ int main() {
       }
     }
     std::cout << (passed ? "PASSED" : "FAILED") << std::endl;
-    return passed;
+    return !passed;
```

Comment:
do 
```cpp
return passed ? EXIT_SUCCESS : EXIT_FAILURE;
```
not work with regtest?

Reply:
Yup that should work, and would be more intuitive as well. I will make this change, thanks!